ush/fix_precip.sh

```diff
@@ -0,0 +1,13 @@
+   #!/bin/sh
```

Comment:
This script is not used in gfs post process.

Reply:
Removed it